.prettierignore

```diff
@@ -8,10 +8,10 @@ node/
 .git
 
 # libraries / external deps / generated files
-war/src/main/js/plugin-setup-wizard/bootstrap-detached.js
+src/main/js/plugin-setup-wizard/bootstrap-detached.js
```

Comment:
Part of the Webpack build invoked from the root Maven module, so the source file moves to the root Maven module.